### DISCUSSION TITLE
Why GitVersion does not work correctly with Git Bash, Cannot find the .git directory

### DISCUSSION THREAD

**Discussion by AlBannaTechno** [GitTools/GitVersion#2689] [Q&A] [answered]
**Hi**,
I just prefer to write this issue here instead of issues, to not overflow the issues section,
because maybe I understand the steps in the wrong way,

anyway,
when I execute  `gitversion /showconfig` from [GitBash](https://git-scm.com/downloads) shipped with Git For Windows, I got this message
`Cannot find the .git directory`

but when I execute it in windows command prompt it just works fine,,

Notice, when I execute `gitversion` without any parameters from `Git Bash`, I get the correct result for the current state of the repository,

So is this an issue, or I did something wrong ?

**Comment by asbjornu** [comment, marked as answer]
In POSIX environments `/` is a path separator, so `/showconfig` is interpreted as the path to a directory in the root called `showconfig`. Try `-showconfig` instead.

**Reply by AlBannaTechno** [reply]
thank you very much, it works, 👍 , I forget that Git Bash is a POSIX.
